test/check-application

```diff
@@ -334,12 +334,12 @@ class TestApplication(testlib.MachineCase):
 
         if auth:
             # Run two containers as system (first exits immediately)
-            self.execute(auth, "podman run -d --name test-sh-system alpine sh")
-            self.execute(auth, "podman run -d --name swamped-crate-system busybox sleep 1000")
+            self.execute(auth, "podman run -d --name test-sh-system alpine:latest sh")
```

Comment:
Ugh, that smells like a podman compat regression? I all of docker's and podman's history, `:latest` has been the implied default if you did not specify a tag. Should we perhaps adjust the tests instead?

Reply:
hmm, I wrote comment in this PR before explaining what is happening. It seems that github ate it, so here we go again:

In the new version, container's `image` property now has the same value, which was used when `podman run` was used. So if you do `podman run busybox` the image will be `busybox` and if you do `podman run busybox:latest` it will be `busybox:latest`. This is nice, as when you have for example `fedora:latest` and `fedora:33` pointing to the same image, and you do `podman run fedora:33` it would show before `fedora:latest`. Now it will show `fedora:33`. Of course, when it assumes `:latest` as it does when you don't specify any tag it could show that as well, but I am also fine with just showing whatever user provided.

Reply:
Ack, that makes sense, thanks!